pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -197,6 +197,7 @@ const (
 // request and that a consumer may accept and allow the service provider access to.
 //
 // +kubebuilder:validation:XValidation:rule="(has(self.all) && self.all) != (has(self.resourceSelector) && size(self.resourceSelector) > 0)",message="either \"all\" or \"resourceSelector\" must be set"
+// +kubebuilder:validation:XValidation:rule="!has(self.group) || self.group != \"tenancy.kcp.dev\" || self.resource != \"logicalclusters\" || (has(self.identityHash) && self.identityHash != \"\")",message="logicalclusters cannot be claimed"
```

Comment:
Would this allow claiming logicalclusters if the identityHash is ""? Looks like it?

Reply:
no, it allows to claim with an identity. And that's fine. It is a different resource then.

Have fixed the api group here.